### PR TITLE
Update and simplify VMware docs

### PR DESCRIPTION
**Pull request by davidcassany**
Removes all the steps related to install Elemental and Cluster creation and focuses only installation media creation and using it to feed a VMware VM.

Fixes rancher/elemental#1557

### CODE REVIEW

**Comment by davidcassany** [comment]
Note most modified files are simply removed images that are no longer displayed as part of the documentation.

File: docs/rancher-vmware.md [hunk 1]
```diff
@@ -7,183 +7,79 @@ title: ''
...
 
-Remove the existing content and enter this code snippet and click on save. Remember to modify this as per your need.
-
-:::info main options
-name: elemental-cls1 --> "Change this as per your need"
-
-device: /dev/sda --> Make sure your target device is "sda" otherwise find out and change the naming convention here, for example in raspberry pi it could be "mmblk---"
-
-emulate-tpm: true  --> Use this only if your target device doesnt have a tpm device and you have a way of emulating tpm like in VMware or KVM
+`emulated-tpm-seed: -1` is sets the client to use a random seed to compute TPM hash, this is useful to be capable
```
Comment (fgiudici) [inline docs/rancher-vmware.md:57]:
```suggestion
`emulated-tpm-seed: -1` sets the client to use a random seed to compute TPM hash, this is useful to be capable
```

File: docs/rancher-vmware.md [hunk 2]
```diff
@@ -7,183 +7,79 @@ title: ''
...
+to reuse the same registration end point definition for multiple machines. See further [TPM documentation](tpm.md).
 
-emulated-tpm-seed: 1 --> increase this by 1 for every new machine. --> If this is not given each machine will recieve the same tpm hash so at any point in time regardless of how many machines you boot it wont show up under the machine inventory.
-:::
+## Step 2: Create the installation ISO
 
-:::danger attention
-Emulate TPM is only for non-production usage like for testing as it beats the purpose of security. So in production usage use the code above without the emulate-tpm and emulate-tpm-seed
-:::
+The installation media needs to be tied to an specific registration end point. This is created and handled
```
Comment (fgiudici) [inline docs/rancher-vmware.md:62]:
```suggestion
The installation media needs to be tied to a specific registration end point. This is created and handled
```

File: docs/rancher-vmware.md [hunk 3]
```diff
@@ -7,183 +7,79 @@ title: ''
...
+with the [SeedImage](seedimage-reference.md) resource.
 
-Once you create the machine registration end point it should show up as active.
+Consider follwing example:
```
Comment (fgiudici) [inline docs/rancher-vmware.md:65]:
```suggestion
Consider the following example:
```

File: docs/rancher-vmware.md [hunk 4]
```diff
@@ -7,183 +7,79 @@ title: ''
...
 
-![Machine registered in Registration Endpoints](images/rancher-vmware-machine-registered.png)
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedImage}</CodeBlock>
 
-Next click on it to view the registration url and download the initial-registration yaml.
+Once the SeedImage resources is created it starts building an ISO with the provided OS image and linking it to
```
Comment (fgiudici) [inline docs/rancher-vmware.md:69]:
```suggestion
Once the SeedImage resource is created it starts building an ISO with the provided OS image and linking it to
```

File: docs/rancher-vmware.md [hunk 5]
```diff
@@ -7,183 +7,79 @@ title: ''
...
-## Part 3: Create the ISO
-
-Make sure you have initial-registration.yaml in the system where you will create the iso in this we will use a Linux vm.
-
-Create a directory to keep everything:
-
-```shell showLineNumbers
-mkdir /home/tux/elemental-demo && cd /home/tux/elemental-demo
+```shell
+wget --no-check-certificate $(kubectl get machineregistration -n fleet-default fire-nodes -o jsonpath="{.status.registrationURL}") -O initial-registration.yaml
```
Comment (fgiudici) [inline docs/rancher-vmware.md:75]: Here we need the link to the SeedImage ISO link, right?

```suggestion
wget --no-check-certificate --content-disposition $(kubectl get seedimages.elemental.cattle.io -n fleet-default fire-img -o jsonpath="{.status.downloadURL}")
```

**Review by fgiudici — CHANGES_REQUESTED** [review]
Nice rework, was needed indeed!
Few typos, just reported on the non-versioned doc.
The cmdline for downloading the ISO is wrong, we need to fix it before merging.

**Comment by davidcassany** [comment]
> Nice rework, was needed indeed! Few typos, just reported on the non-versioned doc. The cmdline for downloading the ISO is wrong, we need to fix it before merging.

Thanks! Good catches, I am adapting the other versions :+1:

File: docs/rancher-vmware.md (hunk 5 above)
Comment (davidcassany) [reply docs/rancher-vmware.md:75]: Indeed I copied the command for the quick start guide and completely forgot to adapt it to this specific use case 😭

**Review by fgiudici — APPROVED** [review]
Looks good!

**Pull request merged**